From dadae4af0da246f6252006d8c4afb9241b9c973a Mon Sep 17 00:00:00 2001
From: Helmut Pozimski <helmut@pozimski.eu>
Date: Wed, 6 Nov 2019 09:10:51 +0100
Subject: [PATCH] Revert "Leverage distutil's build_extensions so that
 --parallel works on Python 3 (GH-3187)"

This reverts commit dfc582d290c9e1c938cc8d2cc1261bbde3a03e05.
---
 Cython/Distutils/old_build_ext.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Cython/Distutils/old_build_ext.py b/Cython/Distutils/old_build_ext.py
index 882f72666..39b6cfb09 100644
--- Cython/Distutils/old_build_ext.py
+++ Cython/Distutils/old_build_ext.py
@@ -191,8 +191,7 @@ class old_build_ext(_build_ext.build_ext):
 
         for ext in self.extensions:
             ext.sources = self.cython_sources(ext.sources, ext)
-        # Call original build_extensions
-        _build_ext.build_ext.build_extensions(self)
+            self.build_extension(ext)
 
     def cython_sources(self, sources, extension):
         """
-- 
2.23.0

